common/djangoapps/pipeline_mako/templates/static_content.html

```diff
@@ -131,21 +125,45 @@
         Uses the Django template engine because our webpack loader only provides template tags for Jinja and Django.
     </%doc>
     <%
-        from django.template import Template, Context
         from webpack_loader.exceptions import WebpackLoaderBadStatsError
-        return Template("""
-            {% load render_bundle from webpack_loader %}
-            {% render_bundle entry %}
-            {% if body %}
-              <script type="text/javascript">
-                  {% autoescape off %}{{ body }}{% endautoescape %}
-              </script>
-            {% endif %}
-        """).render(Context({
-            'entry': entry,
-            'body': capture(caller.body)
-        }))
+        from webpack_loader.templatetags.webpack_loader import render_bundle
```

Comment:
Changing this syntax somehow caused the python tests to stop picking up the `webpack_loader` noop override `INSTALLED_APP` I set up [in this PR](https://github.com/edx/edx-platform/pull/15607/files). Feeling like it's related to how we're now invoking those methods directly instead of as template tags.